client/src/commands/UserInputUtil.ts

```diff
@@ -58,22 +58,18 @@ export class UserInputUtil {
             placeHolder: prompt
         };
 
-        const managedRuntimes: Array<FabricGatewayRegistryEntry> = [];
-        if (showManagedRuntimes) {
-            // Allow users to choose from managed runtimes
-            const runtimeRegistryManager: FabricRuntimeRegistry = FabricRuntimeRegistry.instance();
-            const allRuntimes: FabricRuntimeRegistryEntry[] = runtimeRegistryManager.getAll();
-            for (const runtime of allRuntimes) {
-                const connection: FabricGatewayRegistryEntry = new FabricGatewayRegistryEntry();
-                connection.name = runtime.name;
-                connection.managedRuntime = true;
-                managedRuntimes.push(connection);
-            }
+        const allGateways: Array<FabricGatewayRegistryEntry> = [];
+
+        let connection: FabricGatewayRegistryEntry;
+        if (showManagedRuntime) {
+            // Allow users to choose local_fabric
+            connection = new FabricGatewayRegistryEntry();
+            connection.name = 'local_fabric';
```

Comment:
can we hard code the name of the runtime in the runtime class so if we change the name in the future it will be easier